packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -2066,6 +1980,8 @@ export class CodeModelBuilder {
     (objectSchema as CrossLanguageDefinition).crossLanguageDefinitionId =
       type.crossLanguageDefinitionId;
     this.codeModel.schemas.add(objectSchema);
+    // TODO haoling: fix as any
+    objectSchema.usage = processSchemaUsageFromSdkType(type, objectSchema.usage) as any;
```

Comment:
`objectSchema.usage`'s type is `@autorest\codemodel`'s `SchemaContext` type: packages\http-client-java\node_modules\@autorest\codemodel\dist\model\common\schemas\usage.d.ts
While `processSchemaUsageFromSdkType(type, objectSchema.usage)` returned usage is our `SchemaContext`: packages\http-client-java\emitter\src\common\schemas\usage.ts

The two models are different, hence I use `as any` here to convert first. The ideal way may be create another `ObjectSchema` type in our emitter which extends `@autorest/codemodel`'s `ObjectSchema` and change `ObjectSchema.usage` as our emitter's `SchemaContext` type. But not sure if it is too complex.